lib/lita/handlers/jenkins.rb

```diff
@@ -1,6 +1,13 @@
 require 'json'
 require 'base64'
 
+# Determine if a given String is contains only a number.
+class String
```

Comment:
Good note, I'll take a look at refinements and see if I can figure it out first, otherwise I'll see about sticking it into a module.